src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -342,3 +423,384 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
     # of the maximum PPA boundary created by the Reference Model.
     self.assertTrue(utils.PolygonsAlmostEqual(test_harness_ppa_geometry,
                                               uut_ppa_geometry))
+
```

Comment:
I'm assuming that there are no changes above this line (except `assertPpaCreationFailure()`); please let me know if that is incorrect.

Reply:
Yes you   are right, there is changes except assertPpaCreationFailure())     We have rebased this file to exact difference with master now